_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,308 @@
+---
+id: c249d5
+name: Device motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that it is possible to disable functionality that can be operated by device motion.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation 1
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] is available to disable the event.
```

Comment:
What does it mean to "disable the event'? That seems ambiguous to me. Is it to have the event cancelled, remove the event handler, prevent some effect from occurring in reaction to the event?

Reply:
Rephrased the expectation (without using the term disable and focusing on the outcome of the event) and added a note.